core/chains/evm/config/toml/defaults/BOB_Mainnet.toml

```diff
@@ -0,0 +1,27 @@
+ChainID = '60808'
+# OP stack https://docs.gobob.xyz/learn/introduction/op-stack
+ChainType = 'optimismBedrock'
+# finality_depth was: ~850
+FinalityDepth = 900
```

Comment:
This applies to multiple chains:
When increasing the FinalityDepth from the default, we should also increase the `HeadTracker.HistoryDepth` from its default. The value should be [at least as large](https://docs.chain.link/chainlink-nodes/v1/node-config#historydepth) as FinalityDepth, we've been mostly setting it to 2x.

Reply:
you are 100% right, did't see that, thanks for pointing it out.
Although a small check seems (per other configs) either the same value as the `FinalityDepth` or a bit above it. 

What should I do @mohamed-mehany ? put a 50% more than the FinalityDepth per chain?

Reply:
internal discussions seems that `HistoryDepth` could be good enough using default values, or this could be improved later on. 
Closing this comment for the time being (please re open if still valid)